tests/test_data/pinned.json

```diff
@@ -1,16 +1 @@
-{
-  "repositories": {
-    "imagerepo": {
-      "mirrors": ["http://localhost:30301"]
-    },
-    "director": {
-      "mirrors": ["http://localhost:30401/111"]
-    }
-  },
-  "delegations": [
-    {
-      "paths": ["*"],
-      "repositories": ["imagerepo", "director"]
-    }
-  ]
-}
\ No newline at end of file
+{"delegations":[{"paths":["*"],"repositories":["imagerepo","director"]}],"repositories":{"director":{"mirrors":["file:////home/shikhar/Desktop/uptane/tests/test_data/temp_test_primary/director"]},"imagerepo":{"mirrors":["file:////home/shikhar/Desktop/uptane/tests/test_data/temp_test_primary/imagerepo"]}}}
```

Comment:
Yep, totally agree. I just remembered it was the test_secondary that needed a pinned.json file to function (pointed by travis CI). Working on making test_secondary dynamic right now.
Thanks!